egov-indexer/case-search-indexer.yml

```
@@ -0,0 +1,58 @@
+  mappings:
+    - topic: case-search-topic
+      configKey: INDEX
+      indexes:
+        - name: case-search-index
+          type: general
+          id: $.filingNumber
+          isBulk: false
+          timeStampField: $.filingDate
+          jsonPath: $
+          customJsonMapping:
+            indexMapping: { "Data": { "caseDetails": {}} }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)